core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -502,5 +504,13 @@ class WorldScreen(val viewingCiv:CivilizationInfo) : CameraStageBaseScreen() {
             displayTutorial(Tutorial.Embarking)
     }
 
+    private fun exitGamePrompt() {
+        var promptWindow = PopupTable(this)
+        promptWindow.add("Do you want to exit the game?".tr())
+        promptWindow.row()
+        promptWindow.addButton("Yes"){game.exitEvent?.invoke()}
```

Comment:
Under what circumstance would exitEvent be false? When running from (future) IOS? In that case we shouldn't even add the prompt, no?
Also, if the user clicks the back button multiple times, it will currently generate several popups like this - we should probably try and avoid that (and yes, users will do so, and complain about it)

Reply:
Yes, `exitEvent ` will be null for iOS since there is no Back button, so they have no technical opportunity to call this.
I agree that's a good idea to add a check for multiple Back button clicks.

Reply:
Also in Windows there's no opportunity, and yet you added a close function for Windows. Am I missing something?

Reply:
Desktop, that is

Reply:
That's true, I made an assumption there can be Windows on mobile devices which might have a Back button. But I agree that's quite a rare case.